initoverlayfs.spec

```diff
@@ -1,14 +1,15 @@
 Name:          initoverlayfs
 Version:       0.96
-Release:       1%{?dist}
+Release:       2%{?dist}
 Summary:       An initial scalable filesystem for Linux operating systems
 
-License:       GPLv2
-URL:           https://github.com/ericcurtin/initoverlayfs
+License:       GPL-2.0-only
+URL:           https://github.com/containers/initoverlayfs
 Source0:       %{url}/archive/%{version}/%{name}-%{version}.tar.gz
 
 BuildRequires: gcc
 Recommends: erofs-utils
+Requires: dracut
```

Comment:
This is a "Requires" as of today, I hope to make this a Recommends in future, there's no reason we couldn't use something like https://github.com/systemd/mkosi-initrd instead in future if someone wanted to use that as an alternative tool.

Reply:
I put it in so that a warning on directories owned by the dracut package /usr/lib/dracut/modules.d/ but expected to be there by this package. But if that part goes away or needs to go into a sub package in the future, that is easy enough.